### DISCUSSION TITLE
Sim Card from sysmocom

### DISCUSSION THREAD

**Discussion by Gotsche43** [srsran/srsRAN_Project#580] [General Help] [answered]
I have bought Sysmocom SJA5

https://sysmocom.de/products/sim/sysmoisim-sja5/index.html

I can see a lot of the tutorials only include SJA2.

Is it possible for me to have a functional network using SJA5 with srsRAN and Open5GS?

Regards

**Comment by pgawlowicz** [comment, marked as answer]
Yes, it should be possible.
